README.md

```diff
@@ -5,79 +5,40 @@ RFC](https://github.com/emberjs/rfcs/blob/master/text/0010-engines.md). Engines
 applications to be composed together into a single application from the user's
 perspective.
 
-# Table of Contents
+## Compatibility
 
-**Basic Information**
-
-- [What are Engines?](http://ember-engines.com/guide/what-are-engines)
-- [How does it work?](http://ember-engines.com/guide/core-concepts)
-
-**Usage**
-
-- [Installation](#installation)
-
-**Core Feature Guides**
-
-- [Create an Engine](http://ember-engines.com/guide/creating-an-engine)
-- [Mounting An Engine](http://ember-engines.com/guide/mounting-engines)
-  - [Linking And External Links](http://ember-engines.com/guide/linking-and-external-links)
-  - [Sharing Services](http://ember-engines.com/guide/linking-and-external-links)
-  - [Linking And External Links](http://ember-engines.com/guide/linking-and-external-links)
-  - [Sharing Components and More](http://ember-engines.com/guide/linking-and-external-links)
-  - [Lazy Loading](http://ember-engines.com/guide/lazy-loading)
-- [Engines & Addons](http://ember-engines.com/guide/addons)
-- [Deploying and Fastboot](http://ember-engines.com/guide/deploying-and-fastboot)
-- [Testing](http://ember-engines.com/guide/testing)
-
-**Other Resources**
-
-- [Engine API Documentation](https://api.emberjs.com/ember/release/classes/Engine)
-- [Engines 1.0 Roadmap](https://discuss.emberjs.com/t/engines-1-0-roadmap/14914)
-
-## Introduction Video
-
-[![Introduction to Ember Engines at Global Ember Meetup](https://i.vimeocdn.com/video/559400541_640x360.jpg)](https://vimeo.com/157688181)
```

Comment:
Why delete these links?

Reply:
Because we're linking the Official guides. Like https://github.com/emberjs/ember.js already did.